Cannot import module

It seems like the latest package version wasn't fully published. Running `yarn add --dev esbuild-plugin-sass` (at 0.5.1) and importing the package with `require('esbuild-plugin-sass')` results in the following error:
```
internal/modules/cjs/loader.js:888
  throw err;
  ^

Error: Cannot find module './internals/external'
Require stack:
- /[projectDir]/node_modules/esbuild-plugin-sass/index.js
- /[projectDir]/scripts/serve.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:885:15)
    at Function.Module._load (internal/modules/cjs/loader.js:730:27)
    at Module.require (internal/modules/cjs/loader.js:957:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (/[projectDir]/node_modules/esbuild-plugin-sass/index.js:2:20)
    at Module._compile (internal/modules/cjs/loader.js:1068:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1097:10)
    at Module.load (internal/modules/cjs/loader.js:933:32)
    at Function.Module._load (internal/modules/cjs/loader.js:774:14)
    at Module.require (internal/modules/cjs/loader.js:957:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/[projectDir]/node_modules/esbuild-plugin-sass/index.js',
    '/[projectDir]/scripts/serve.js'
  ]
}
```
Indeed, the only files in the `internals` dir are `external.d.ts` and `external.d.ts.map`.

--- a/package.json
+++ b/package.json
@@ -64,7 +64,7 @@
     "index.js",
     "index.d.ts",
     "index.d.ts.map",
-    "internals/*.d.js",
+    "internals/*.js",
     "internals/*.d.ts",
     "internals/*.d.ts.map"
   ]

